app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java

```diff
@@ -101,30 +84,8 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         buttonSensorAutoScan.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (scienceLab.isConnected()) {
-                    try {
-                        data = i2c.scan(null);
-                    } catch (IOException e) {
-                        e.printStackTrace();
-                    }
-
-                    if (data != null) {
-                        for (Integer myInt : data) {
-                            if (myInt != null && sensorAddr.get(myInt) != null) {
-                                dataAddress.add(String.valueOf(myInt));
-                                dataName.add(sensorAddr.get(myInt));
-                            }
-                        }
-                        tvData = "";
-                        for (String s : dataAddress) {
-                            tvData += s + ":" + sensorAddr.get(Integer.parseInt(s)) + "\n";
-                        }
-
-                    }
-                    tvSensorScan.setText(tvData);
-                    String[] dataDisp = dataName.toArray(new String[dataName.size()]);
-                    adapter = new ArrayAdapter<>(getContext(), R.layout.sensor_list_item, R.id.tv_sensor_list_item, dataDisp);
-                    lvSensor.setAdapter(adapter);
+                if (!scienceLab.isConnected()) {
```

Comment:
I think , i need to make another PR , because , the updation done by merging the master has removed my code.